zwave_js_server/client.py

```diff
@@ -55,8 +54,16 @@ def connected(self) -> bool:
         """Return if we're currently connected."""
         return self._client is not None and not self._client.closed
 
-    async def async_send_command(self, message: Dict[str, Any]) -> dict:
+    async def async_send_command(
+        self,
+        message: Dict[str, Any],
+        require_scheme: Optional[int] = None,
+    ) -> dict:
         """Send a command and get a response."""
+        if require_scheme is not None and require_scheme > self.scheme_version:
+            raise InvalidServerVersion(
+                "This command is not available, please update Z-Wave Server to a newer version."
```

Comment:
Include what version is required.

Reply:
we require a server with supported scheme version, so not a specific version itself.
Should we log that ?

Reply:
The minimum required version I mean.

Reply:
I've adjusted the message